wp-content/themes/core/components/blocks/gallery_grid/gallery_grid.php

```diff
@@ -0,0 +1,64 @@
+<?php
+declare( strict_types=1 );
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = \Tribe\Project\Templates\Components\blocks\gallery_grid\Gallery_Grid_Controller::factory( $args );
+?>
+
+<section <?php echo $c->get_classes(); ?> <?php echo $c->get_attributes(); ?>>
+	<div <?php echo $c->get_container_classes(); ?>>
+
+		<div class="b-gallery-grid__header">
+			<?php if ( ! empty( $c->get_header_args() ) ) { ?>
+				<?php get_template_part(
+					'components/content_block/content_block',
+					null,
+					$c->get_header_args()
+				); ?>
+			<?php } ?>
+
+			<?php if ( $c->is_slideshow() ) : ?>
+				<div class="b-gallery-grid__cta">
+					<?php get_template_part( 'components/button/button', null, $c->get_slideshow_button() ); ?>
+				</div>
+			<?php endif; ?>
+		</div>
+
+		<div <?php echo $c->get_content_classes(); ?>>
+			<?php foreach ( $c->get_gallery_img_thumbs() as $img ) {
+				get_template_part(
+					'components/image/image',
+					null,
+					$img
+				);
+			} ?>
+		</div>
+	
+	</div>
+</section>
+
+<?php if ( $c->is_slideshow() ) : ?>
+	<script data-js="dialog-content-<?php echo $c->get_block_id(); ?>" type="text/template">
+		<div class="c-dialog">
+			<div class="c-dialog__overlay">
+				<div class="c-dialog__header">
+					<div class="c-dialog__title"><?php echo $c->get_slideshow_title(); ?></div>
```

Comment:
Should the dialog title be a heading tag? Additionally, I think this should all be moved to a dialog component's template, and controller and rendered via a `get_template_part()` rather than hard coded in this component's template.